packages/teuchos/parameterlist/test/ParameterList/ParameterEntry_UnitTest.cpp

```diff
@@ -45,6 +45,47 @@
 
 namespace Teuchos {
 
+//! @test Check that the move constructor of @ref Teuchos::ParameterEntry works as expected.
+TEUCHOS_UNIT_TEST(Teuchos_ParameterEntry, move_constructor)
+{
+    ParameterEntry source(std::string("nice-entry"));
+    TEST_EQUALITY(Teuchos::any_cast<std::string>(source.getAny()),"nice-entry");
```

Comment:
Teuchos indents 2 spaces not 4 spaces.  It is good to be consistent.

Reply:
Done :heavy_check_mark: